satellite/audit/reporter.go

```diff
@@ -167,6 +174,16 @@ func (reporter *Reporter) recordAuditSuccessStatus(ctx context.Context, successN
 			IsUp:         true,
 			AuditSuccess: true,
 		}
+
+		notification := &pb.Notification{
+			Scope:     pb.Notification_AUDIT,
+			Level:     pb.Notification_INFO,
+			Tags:      nil,
+			Message:   "Successfully passed audit",
+			Timestamp: time.Now(),
+		}
+
+		reporter.notifications.Notify(ctx, nodeID, notification)
```

Comment:
A slow node could end up blocking audits.

Reply:
that call is async

Reply:
Ah, let's make it clear by naming the method as such.